packages/tooltip/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
We need to update the root `tsconfig.json` to include this package too.

Reply:
👍